app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def index
+    @likes = Like.all
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @user = current_user
+    @like = Like.new(user: @user, post: @post)
+    if @like.save
+      @post.update(likes_counter: @post.likes_counter + 1)
```

Comment:
[OPTIONAL] I think a method like this would be better in the model since it is not directly related to requests. And I think using a tool like countet cache would be better suited since it handles increment and decrement easily.

Reply:
I would apply this suggestion as well. Thanks so much